migrations/002_deterministic_scans.sql

```
@@ -0,0 +1,23 @@
+    mode VARCHAR(50) NOT NULL DEFAULT 'deep',
+    overall_score INTEGER NOT NULL,
+    letter_grade VARCHAR(10),
+    archetype VARCHAR(100),
+    data JSONB NOT NULL,
+    api_calls_used INTEGER DEFAULT 0,
+    duration_ms INTEGER,
+    created_at TIMESTAMPTZ DEFAULT NOW()
+);
+
+CREATE INDEX IF NOT EXISTS idx_deterministic_scans_user_id_created_at 
+    ON deterministic_scans(user_id, created_at DESC);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)